setup.py

```diff
@@ -113,7 +114,20 @@ def get_pybind11_include_path():
 
 def get_cpp_extensions():
     cpp_transform_pkgdir = Path('hexrd') / 'transforms/cpp_sublibrary'
-    src_files = [str(cpp_transform_pkgdir / 'src/inverse_distortion.cpp')]
+    src_files = [
+        str(cpp_transform_pkgdir / 'src/transforms.cpp'),
+        str(cpp_transform_pkgdir / 'src/inverse_distortion.cpp'),
+    ]
+
+    extra_compile_args = [
+        '-O3',
+        '-Wall',
+        '-shared',
+        '-std=c++14',
+        '-funroll-loops',
```

Comment:
I think Windows is just going to straight up ignore these. Maybe we should write alternative flags to use for Windows, so it will be optimized as well? Like we did above on line 47. We do have Windows users.

Reply:
This is a good point, I'll come back to this in a future PR